providers/fab/src/airflow/providers/fab/www/session.py

```diff
@@ -24,22 +24,19 @@
 
 
 class _LazySafeSerializer:
-    def encode(self, session_dict):
+    def dumps(self, session_dict):
         encoder = msgspec.msgpack.Encoder(
             enc_hook=lambda obj: str(obj) if isinstance(obj, LazyString) else obj
         )
-
         return encoder.encode(dict(session_dict))
 
-    def decode(self, data):
+    def loads(self, data):
         decoder = msgspec.msgpack.Decoder()
-
         return decoder.decode(data)
 
-    def _default(self, obj):
```

Comment:
Mostly just curious: Why did we drop this default?

Reply:
Unused